src/chi/themes/test/_variables.scss

```diff
@@ -1323,23 +1329,35 @@ $mobile-nav-list-item-active-box-shadow: inset 0.125rem 0 0 $color-border-primar
 
 // Modal
 $modal-animated-transition-duration: 0.5s;
-$modal-background-color: $color-grey-10;
-$modal-border-radius: $border-radius-base;
-$modal-box-shadow: 0 24px 38px 3px rgba(0, 0, 0, 0.14), 0 9px 46px 8px rgba(0, 0, 0, 0.12),
-  0 11px 15px -7px rgba(0, 0, 0, 0.2);
-$modal-close-top: 0.5rem;
-$modal-max-width: 600px;
-$modal-header-background-color: $color-teal-70;
+$modal-background-color: $color-white;
```

Comment:
@wojciechwlodarczyklumen could we improve it by using corresponding variables as $color-background-white when posible? Same for icons, text...